lib/src/protos/working_copy.proto

```diff
@@ -37,7 +37,10 @@ message SparsePatterns {
 }
 
 message TreeState {
-  bytes tree_id = 1;
+  bytes legacy_tree_id = 1 [deprecated = true];
```

Comment:
nit: I don' think `legacy_tree_id` is deprecated yet. We still use the legacy tree by default.

Reply:
That's fair :) Dropped the annotation and the associated `#[allow(deprecated)]`.